cargo-pgrx/src/command/init.rs

```diff
@@ -70,6 +70,13 @@ pub(crate) struct Init {
     base_testing_port: Option<u16>,
     #[clap(long, help = "Additional flags to pass to the configure script")]
     configure_flag: Vec<String>,
+    /// Compile PostgreSQL with the necessary flags to detect a good amount of
+    /// memory errors when run under Valgrind.
+    ///
+    /// Building PostgreSQL with these flags requires that Valgrind be
+    /// installed, but the resulting build is usable without valgrind.
+    #[clap(long)]
+    valgrind: bool,
```

Comment:
I don't understand why these two clauses are simultaneously true ?_?

I suppose it's not critical that I do, but

Reply:
Valgrind must be installed because postgres does something like `#include <valgrind.h>` to bring some macros into scope. Those macros expand to asm statements that are essentially just spicy nops when valgrind is not connected.

Reply:
Oh, I meant "why is it still slow"?

Reply:
I edited. But it's probably slow due to the -Og which might actually be unnecessary given that we aren't debugging Postgres, just the extensions.

Reply:
Oh okay!

Reply:
I documented this, and removed -Og